testsuite/integration-arquillian/pom.xml

```diff
@@ -40,8 +40,8 @@
         <app.server>undertow</app.server>
 
         <!-- Wildfly deprecated versions -->
-        <wildfly.deprecated.version>22.0.1.Final</wildfly.deprecated.version>
-        <wildfly.deprecated.wildfly.core.version>14.0.1.Final</wildfly.deprecated.wildfly.core.version>
+        <wildfly.deprecated.version>23.0.0.Final</wildfly.deprecated.version>
```

Comment:
I'm not sure if we should put here the second latest available version or rather previous version which was released within a Keycloak release. Maybe @vramik can clarify this.

Reply:
This needs to be decided by @pdrozd. I believe there should be either WF22 or WF21. I'm not sure because latest released keycloak is based on WF21, but we're upgrading to WF23 so we've skipped WF22. It'd mean we won't test WF22 at all.

Reply:
@pdrozd Hey Pavel, -^ pls let me know which version (WF22 or WF21) do you prefer there?

Thanks,
Jan

@martin-kanis @vramik Thanks for looking / your suggestions too!

Reply:
@iankko Sorry for the late reply. After a discussion with @vramik, lets put a WF21 here.

Reply:
@pdrozd Thanks for the clarification, Pavel Returned their versions back (see last commit) to the state they had in Wildfly 21 (`21.0.2.Final`) state: 
  https://github.com/keycloak/keycloak/pull/7857/commits/eadb03f227c8f1f9673f267493ae35e7fbc2dfbe#diff-3f11c230f74300e6104be0c244f27eceb03e297ba46b2bea29086c95a9eb5dc6L43

Please double-check.

Thanks, Jan